src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/language/LocalLanguageDetector.scala

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.partnercatalyst.fortis.spark.transforms.language
+
+import com.microsoft.partnercatalyst.fortis.spark.logging.FortisTelemetry
+import com.optimaize.langdetect.LanguageDetectorBuilder
+import com.optimaize.langdetect.ngram.NgramExtractors
+import com.optimaize.langdetect.profiles.LanguageProfileReader
+import com.optimaize.langdetect.text.CommonTextObjectFactories
+
+@SerialVersionUID(100L)
+class LocalLanguageDetector extends LanguageDetector {
+  @transient private lazy val languageProfiles = new LanguageProfileReader().readAllBuiltIn
+  @transient private lazy val languageDetector = LanguageDetectorBuilder.create(NgramExtractors.standard()).withProfiles(languageProfiles).build()
+  @transient private lazy val textObjectFactory = CommonTextObjectFactories.forDetectingOnLargeText()
```

Comment:
Is large text always the best choice here? I see the other relevant-sounding option is `forDetectingShortCleanText`. While Tweets probably aren't clean, they're also not long.